user-interface/src/main/java/life/qbic/datamanager/views/projects/qualityControl/QCItemDeletionConfirmationNotification.java

```diff
@@ -0,0 +1,31 @@
+package life.qbic.datamanager.views.projects.qualityControl;
+
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import life.qbic.datamanager.views.notifications.NotificationDialog;
+
+/**
+ * Warns the user that the file will be deleted from the server.
+ * <p>
+ * This dialog is to be shown when a user wants to delete a Quality Control Item.
+ */
+public class QCItemDeletionConfirmationNotification extends NotificationDialog {
+
+  public QCItemDeletionConfirmationNotification() {
+    addClassName("batch-deletion-confirmation");
```

Comment:
It would be nice if the css class is generalized and the class name is adapted for all notifications
```suggestion
    addClassName("deletion-confirmation");
```